lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -170,13 +173,15 @@
     - List of dictionaries describing the service secrets.
     - Every item must be a dictionary exposing the keys secret_id, secret_name, filename, uid (defaults to 0), gid (defaults to 0), mode (defaults to 0o444)
     - Maps docker service --secret option.
+    - Requires api_version >= 1.25
```

Comment:
```suggestion
    - Requires API version >= 1.25
```
`api_version` is not a term that should be used in user-facing documentation (feel free to update the other occurences).